### ISSUE TITLE
Namespaces.UnusedUses confused by multiple open tags

### ISSUE THREAD

**Issue by sebastienbarre** [slevomat/coding-standard#772]
This example below does not trigger `SlevomatCodingStandard.Namespaces.UnusedUses`:
```php
<?php

use View\Helper\FilterHelper;

echo FilterHelper::openFilterContainer();
```

However, as soon as I introduce a new opening tag:
```php
<?php

use View\Helper\FilterHelper;

echo FilterHelper::openFilterContainer();
?>
<?php echo "bar";
```
I get:
```
 3:1  error  phpcs:error  Type View\Helper\FilterHelper is not used in this file.
```

Which is problematic because we do have a ton of old-school view files that mix PHP and HTML, with multiple opening tags throughout.

Am I using this sniff wrong?

At first I suspected that it was only considering the contents of the last piece of PHP code in between an opening tag and the end of the file (which would still be a problem)... but apparently not, as this example below will also trigger the sniff:
```php
<?php

use View\Helper\FilterHelper;

echo FilterHelper::openFilterContainer();
?>
<?php echo FilterHelper::openFilterContainer();
```

Thank you

**Comment by sebastienbarre** [comment]
Hmmm, I'm realizing I'm using the latest release, 5.0.4, from March 22nd, and this sniff had one fix since then, on master, and it looks very much related: 71ad0971a05f557d88cc7747d24d9a83f122bc0f 

I'm going to try to pull master in our project, and close this issue if that's the case. Though a new release would be great, it's been a while :)

**Comment by sebastienbarre** [comment]
Yup, this was resolved by 71ad0971a05f557d88cc7747d24d9a83f122bc0f in master. Thanks to @donatj. Closing this, sorry for the noise. If someone else bumps into it, make sure to use master, not 5.0.4.

**Comment by donatj** [comment]
@sebastienbarre Yup. I have been very patiently awaiting a tagged release since April ;)
